src/main/resources/templates/shift-info.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Excel Parsing Web App</title>
+    <link th:href="@{main.css}" rel="stylesheet"/>
+</head>
+<body>
+<span th:text="${msg}"></span>
+<h2>Select Shift information for each officer</h2>
```

Comment:
it's a good idea to observe the hierarchy of headers. You don't have header 1. If you don't like its style, you can alays change it using css.

Reply:
fixed